tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -973,6 +986,10 @@ def test_list_recommendations_with_changes_in_update_results(cluster_type: str):
 
             short_term_recommendation = data_section[str(end_time)]["duration_based"]["short_term"]
 
+            assert "variation" in short_term_recommendation
```

Comment:
@bharathappali - As discussed, variations validation is common code and should be included in utils.py in validate_reco_json.  And value for amount needs to be validated in variation as below:

variation = recommended - current (if requests/limits is set)
variation = recommended (if requests/limits is not set)

Reply:
Added the generic check in the utils